sample_files/.travis.yml

```diff
@@ -27,4 +30,4 @@ script:
   - travis_run_tests
 
 after_success:
-  coveralls
```

Comment:
I agree with adding an after success script. It gives us a nice extension point for future needs.